RELEASE.md

```diff
@@ -0,0 +1,38 @@
+# Release
+
+* Update [CHANGELOG.md](https://github.com/lektor/lektor-atom/blob/master/CHANGELOG.md)
+
+* Update package version on [setup.py](https://github.com/lektor/lektor-atom/blob/master/setup.py)
+
+* Commit changes
+  ```bash
+  git add .
+  git commit -m "Prepare release version"
+  ```
+
+* Tag the release
+  ```bash
+  git tag -a vX.X.X -m 'Release version'
+  ```
+
+* Push changes
+  ```bash
+  git push origin master
+  git push origin --tags
+  ```
+
+* The new tag will trigger a Github action workflow on CI and will create [pre release](https://github.com/lektor/lektor-atom/releases).
+
+* Once that release is moved from prerelease to release another Github Action will publish to PyPI using the `getlektor` account.
+
+## Retrigger events
+
+If the release process needs to be fixed or retriggered you will need to delete the tag and recreate it, or create a new tag from scratch.
+
+* To delete and recreate a tag:
+  ```bash
+  git tag -d <tag-name>
+  git push origin :refs/tags/<tag-name>
```

Comment:
This still works, but there is a more modern explicit option too: `git push --delete origin refs/tags/<tag-name>`.

Reply:
Oh la la! Good one :-)